tests/vmi_networking_test.go

```diff
@@ -895,6 +895,13 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 
 					return nil
 				}, 120*time.Second).Should(Succeed())
+
+				By("Restarting the vmi")
+				Expect(console.ExpectBatch(vmi, []expect.Batcher{
```

Comment:
Can you use `RunCommand` instead?
If you prefer the "safe" version, then `SafeExpectBatch` fits as well.

Reply:
"sudo shutdown -r now\n" doesn't return a prompt.

`SafeExpectBatch` cannot be used since it uses `ExpectBatchWithValidatedSend` which [requires at least 2 batchers](https://github.com/kubevirt/kubevirt/blob/f47a643e91eadf2431e320ada9e74a9852e29d3c/tests/console/console.go#L236).
`RunCommand` cannot be used as well since it [waits for the](https://github.com/kubevirt/kubevirt/blob/f47a643e91eadf2431e320ada9e74a9852e29d3c/tests/console/console.go#L112) `PromptExpression`.